### PR TITLE
Fat2 134 sorting

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**

### CODE REVIEW

File: src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs [hunk 1]
```diff
@@ -27,15 +28,16 @@ public class TrainingCoursesController : ControllerBase
         }
 
         [HttpGet]
-        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null)
+        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null, [FromQuery]int orderBy = 0)
```
Comment (scottywakefield) [inline src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs:31]: web is sending this as a string. semantic web and all.

File: src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs [hunk 2]
```diff
@@ -27,15 +28,16 @@ public class TrainingCoursesController : ControllerBase
...
         {
             try
             {
                 var queryResult = await _mediator.Send(new GetTrainingCoursesListQuery
                 {
                     Keyword = keyword, 
                     RouteIds = routeIds,
-                    Levels = levels
+                    Levels = levels,
+                    OrderBy = (OrderBy)orderBy
```
Comment (scottywakefield) [inline src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs:40]: having a string from front end can be mapped to the web's enum, and then map to inner api enum. don't assume that they will map 0==0, 1==1. be explicit. else there'll be mysterious bugs one day when a new order by is added or removed.

File: src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetStandardsListRequest.cs [hunk 3]
```diff
@@ -15,6 +14,7 @@ public class GetStandardsListRequest : IGetApiRequest
         public string GetUrl => BuildUrl();
 
         public List<int> Levels { get ; set ; }
+        public OrderBy OrderBy { get; set; }
```
Comment (scottywakefield) [inline src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetStandardsListRequest.cs:17]: won't you need that in build url as well?

File: src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs]: This is fixed

File: src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs]: This is also fixed

File: src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetStandardsListRequest.cs (hunk 3 above)
Comment (dashton82) [reply src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetStandardsListRequest.cs:17]: Updated to be in the request to the inner api

**Pull request merged**